cli/factory.rs

```diff
@@ -300,7 +301,13 @@ impl CliFactory {
       .npm_resolver
       .get_or_try_init_async(async {
         let fs = self.fs();
-        create_cli_npm_resolver(
+        create_cli_npm_resolver(if self.options.unstable_byonm() {
+          CliNpmResolverCreateOptions::Byonm(CliNpmResolverByonmCreateOptions {
+            fs: fs.clone(),
+            // todo(byonm): actually resolve this properly
```

Comment:
Why this solution is not proper?

Reply:
I will update the comment. It's because the package.json could be in an ancestor directory and not the initial cwd.